### PR TITLE
Update more demos to DJL 0.20.0

### PR DESCRIPTION
**Pull request by xyang16**

### CODE REVIEW

File: apache-beam/ctr-prediction/pom.xml [hunk 1]
```diff
@@ -244,7 +244,7 @@
         </dependency>
         <dependency>
             <groupId>ai.djl.mxnet</groupId>
-            <artifactId>mxnet-native-auto</artifactId>
```
Comment (frankfliu) [inline apache-beam/ctr-prediction/pom.xml:247]: Just remove this dependency

File: apache-flink/sentence_embedding/build.gradle [hunk 2]
```diff
@@ -11,13 +11,13 @@ repositories {
...
     implementation 'org.apache.flink:flink-streaming-java_2.11:1.11.1'
-    implementation platform("ai.djl:bom:0.12.0")
+    implementation platform("ai.djl:bom:0.20.0")
     implementation "ai.djl:api"
     implementation "org.slf4j:slf4j-simple:1.7.30"
 
     runtimeOnly 'org.apache.flink:flink-clients_2.11:1.11.1'
     runtimeOnly "ai.djl.tensorflow:tensorflow-model-zoo"
-    runtimeOnly "ai.djl.tensorflow:tensorflow-native-auto"
+    runtimeOnly "ai.djl.tensorflow:tensorflow-native-cpu"
```
Comment (frankfliu) [inline apache-flink/sentence_embedding/build.gradle:20]: remove -auto dependencies

File: apache-flink/sentiment-analysis/build.gradle [hunk 3]
```diff
@@ -11,13 +11,13 @@ repositories {
...
     implementation 'org.apache.flink:flink-streaming-java_2.11:1.11.1'
-    implementation platform("ai.djl:bom:0.12.0")
+    implementation platform("ai.djl:bom:0.20.0")
     implementation "ai.djl:api"
     implementation "org.slf4j:slf4j-simple:1.7.30"
 
     runtimeOnly 'org.apache.flink:flink-clients_2.11:1.11.1'
     runtimeOnly "ai.djl.pytorch:pytorch-model-zoo"
-    runtimeOnly "ai.djl.pytorch:pytorch-native-auto"
+    runtimeOnly "ai.djl.pytorch:pytorch-native-cpu"
```
Comment (frankfliu) [inline apache-flink/sentiment-analysis/build.gradle:20]: same here

File: pneumonia-detection/build.gradle [hunk 4]
```diff
@@ -12,11 +12,11 @@ repositories {
 }
 
 dependencies {
-    implementation platform("ai.djl:bom:0.12.0")
+    implementation platform("ai.djl:bom:0.20.0")
     implementation "ai.djl:api"
 
     runtimeOnly "ai.djl.tensorflow:tensorflow-engine"
-    runtimeOnly "ai.djl.tensorflow:tensorflow-native-auto"
+    runtimeOnly "ai.djl.tensorflow:tensorflow-native-cpu"
```
Comment (frankfliu) [inline pneumonia-detection/build.gradle:19]:
```suggestion
```

File: quarkus/example/pom.xml [hunk 5]
```diff
@@ -158,7 +158,7 @@
                 </dependency>
                 <dependency>
                     <groupId>ai.djl.pytorch</groupId>
-                    <artifactId>pytorch-native-auto</artifactId>
+                    <artifactId>pytorch-native-cpu</artifactId>
```
Comment (frankfliu) [inline quarkus/example/pom.xml:161]: remove

File: quarkus/extension-example/pom.xml [hunk 6]
```diff
@@ -18,7 +18,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.compiler.parameters>true</maven.compiler.parameters>
         <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
-        <djl.version>0.10.0</djl.version>
+        <djl.version>0.20.0</djl.version>
```
Comment (frankfliu) [inline quarkus/extension-example/pom.xml:21]: This won't work anymore, we can either remove this example, or leave it as is

File: web-demo/doodle_game/backend/build.gradle [hunk 7]
```diff
@@ -10,10 +10,10 @@ repositories {
 }
 
 dependencies {
-    implementation platform("ai.djl:bom:0.10.0")
+    implementation platform("ai.djl:bom:0.20.0")
     implementation "ai.djl:api"
     implementation "ai.djl.pytorch:pytorch-engine"
-    implementation "ai.djl.pytorch:pytorch-native-auto"
+    implementation "ai.djl.pytorch:pytorch-native-cpu"
```
Comment (frankfliu) [inline web-demo/doodle_game/backend/build.gradle:16]: remove

File: pneumonia-detection/build.gradle (hunk 4 above)
Comment (xyang16) [reply pneumonia-detection/build.gradle:19]: This is tensorflow-engine dependency, should we keep the native?

File: web-demo/doodle_game/backend/build.gradle (hunk 7 above)
Comment (xyang16) [reply web-demo/doodle_game/backend/build.gradle:16]: This is pytorch-engine dependency, should we keep the native?

File: pneumonia-detection/build.gradle (hunk 4 above)
Comment (frankfliu) [reply pneumonia-detection/build.gradle:19]: We don't have any -auto package. only engine jar is needed

**Review by frankfliu — APPROVED** [review]

File: pneumonia-detection/build.gradle (hunk 4 above)
Comment (frankfliu) [reply pneumonia-detection/build.gradle:19]: No, we don't. all native package need a classifier

Comment (xyang16) [reply pneumonia-detection/build.gradle]: OK

**Pull request merged**